board/batocera/fsoverlay/etc/init.d/S11share

```diff
@@ -81,19 +81,18 @@ mountDeviceOrFallback() {
     fi
 }
 
-systemsetting="batocera-settings"
-rb_wifi_configure() {
+batocera_wifi_configure() {
     X=$1
 
     if test "$X" = 1
     then
-	settings_ssid="`$systemsetting -command load -key wifi.ssid`"
-	settings_key="`$systemsetting -command load -key wifi.key`"
+	settings_ssid="$(batocera-settings -command load -key wifi.ssid)"
+	settings_key="$(batocera-settings -command load -key wifi.key)"
 	settings_file="/var/lib/connman/batocera_wifi.config"
 	settings_name="default"
     else
-	settings_ssid="`$systemsetting -command load -key wifi${X}.ssid`"
-	settings_key="`$systemsetting -command load -key wifi${X}.key`"
+	settings_ssid="$(batocera-settings -command load -key wifi"${X}".ssid)"
+	settings_key="$(batocera-settings -command load -key wifi"${X}".key)"
```

Comment:
The Quoting of ${X} is not needed we are still inside the full quotes.

Reply:
Note that $( ) starts a new context, and variables in it have to be quoted independently.
examples : 

```
echo "This $variable is quoted $(but this $variable is not)" bad 
echo "This $variable is quoted $(and now this "$variable" is too)" good
```